routes/controllers/web-api.js

```diff
@@ -1,23 +1,41 @@
 
+const {pick} = require('lodash');
+
 async function find(req, res) {
-    const {
-        query,
-        property,
-        author,
-        sort,
-        start,
-        amount,
-    } = req.query;
-
-    if (!query && !author && !sort) {
+    const properties = {
+        query: req.query.query,
+        property: req.query.property,
+        author: req.query.author,
+        sort: req.query.sort,
+        start: req.query.start,
+        amount: req.query.amount,
+    };
+
+    // Remove undefined properties
+    Object.keys(properties).forEach((key) => {
+        if (properties[key] === undefined) delete properties[key];
```

Comment:
Then it would perform a search for nothing, which would return no results. I don't think that will ever be requested from the API, but I'll set it to remove that too.